apim-operator/deploy/controller-artifacts/operator.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       containers:
         - name: apim-operator
           # Replace this with the built image name
-          image: wso2am/apim-operator:v0.5
+          image: wso2am/apim-operator:v2.7
```

Comment:
Why a sudden increase? Let's keep v1.0 for the initial release.

Reply:
changes are added